### ISSUE TITLE
Rename `-able` traits to `-Ext`

### ISSUE THREAD

**Issue by bvssvni** [PistonDevelopers/conrod#309]
This is more consistent with the standard library coding guidelines.

I'm planning to do this for piston-graphics https://github.com/PistonDevelopers/graphics/issues/819

**Comment by bvssvni** [comment]
- [ ] `Callable => CallbackExt`
- [ ] `Colorable => ColorExt`
- [ ] `Frameable => FrameExt`
- [ ] `Labelable => LabelExt`
- [ ] `Positionable => PositionExt`
- [ ] `Shapeable => ShapeExt`
- [ ] `Themeable => ThemeExt`

`Drawable` is not duck typed yet, which might be turned into an action `Draw`. If it becomes an action, then `Drawable` could be renamed to `DrawExt`.

**Comment by mitchmindtree** [comment]
Ahh yes, kind of like `Iterator` => `IteratorExt`?

Come to think of it, I guess the names are closer to `Iterator` though, as they don't actually extend any other traits as IteratorExt does. Thoughts?

**Comment by bvssvni** [comment]
Yeah, the convention doesn't fit perfect. Perhaps we should come up without own `Ext`?

**Comment by bvssvni** [comment]
`ThemeDuck` haha

**Comment by bvssvni** [comment]
`Duck` doesn't fit either, because the traits can be implemented without duck types as well.

**Comment by bvssvni** [comment]
What about `Rel`? It has 3 letters like `Ext` and the traits are often used to move an object "relative" to something else.

* `TransformRel`
* `PositionRel`

**Comment by bvssvni** [comment]
I have an idea: What about `Positioned` instead of `Positionable`? It is shorter and sounds better.

- [ ] `Callable => Called`
- [ ] `Colorable => Colored`
- [ ] `Frameable => Framed`
- [ ] `Labelable => Labeled`
- [ ] `Positionable => Positioned`
- [ ] `Shapeable => Shaped`
- [ ] `Themeable => Themed`

It also solves the problem `SourceRectangleable` in piston-graphics which becomes `SourceRectangled`.

**Comment by bvssvni** [comment]
I've edited the comments to focus discussion on `-ed`.
